components/style/properties/gecko.mako.rs

```diff
@@ -3152,6 +3142,14 @@ fn static_assert() {
         self.copy_display_from(other)
     }
 
+    pub fn set_original_display(
```

Comment:
This is still wrong, you don't want to set `mOriginalDisplay` from `StyleAdjuster`, you just want to set `mDisplay`, I'd keep the `set_adjusted_display` name for doing that.

Reply:
Got it. Let me change it back to `set_adjusted_display`.
I was thinking the purpose is to `set original display` so I named it with `set_original_display`.